### DISCUSSION TITLE
Use reusable workflow in the public github from GHE action config.

### DISCUSSION THREAD

**Discussion by hiker0304** [community/community#40430] [Enterprise]
### Select Topic Area

Question

### Body

Hi!
Now, I'm using GHE 3.5
I try to use reusable workflow.
Our repositories are whole private repository.
We can not create public or internal repository in our ORG.
I successfully run the workflow with reusable workflows in the same repository.

But, when I try to run the workflow with reusable workflows in the public github ( github.com ).
it is failed with below error.
```
[Invalid workflow file: .github/workflows/master.yml#L70]
.....
error parsing called workflow "#OWNER/#REPO/.github/workflows/xxxxxx.yml@0575c7e1d3d4a8fbc8c73b9141efa9aa65337a00": workflow was not found. See https://docs.github.com/enterprise-server@3.5/actions/learn-github-actions/reusing-workflows#access-to-reusable-workflows for more information.
```
I created reusable workflow in my account in the public github.

I don't know what the problem is

**Comment by ossanna16** [comment]
Hi there @hiker0304 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
